src/EthMultiVault.sol

```diff
@@ -2167,6 +2337,22 @@ contract EthMultiVault is IEthMultiVault, Initializable, ReentrancyGuardUpgradea
         return address(bytes20(rawAddress << 96));
     }
 
+    /// @notice Check if a sender is approved to deposit on behalf of a receiver
```

Comment:
3.16 - Utilize Helper Functions More

Here, we created some helper methods to check deposit and redeem approval.